suministrospr/suministros/templates/suministros/suministro_list.html

```diff
@@ -1,32 +1,46 @@
 {% extends "base.html" %}
 
 {% load static %}
+{% load i18n %}
 {% load suministro_extras %}
 
 {% block head_title %}Entérate, Ayuda, Informa - {% endblock %}
 
 {% block content %}
 
+{% get_current_language as language_code %}
 <div class="main__about">
   <div class="main__about main__about--paragraph">
     <p>
+     {% if language_code == 'es' %}
       La donación de suministros a las áreas más afectadas del país por el terremoto ha sido un éxito. Esta plataforma, administrada por los mismos donantes y recipientes, busca ser un vehículo de distribución equitativa de estos suministros. De esta manera, la ayuda llega a donde se necesita.
+      {% endif %}
+      {% if language_code == 'en' %}
+      La donación de suministros a las áreas más afectadas del país por el terremoto ha sido un éxito. Esta plataforma, administrada por los mismos donantes y recipientes, busca ser un vehículo de distribución equitativa de estos suministros. De esta manera, la ayuda llega a donde se necesita.
```

Comment:
Here's one option for this translation

```suggestion
      The donation of supplies to the areas affected by the Puerto Rico earthquakes has been a success. This platform, that is administrated by the same donors and recipients, tries to be a fair distribution method for these supplies. This way, supplies arrive where they are most needed.
```